alerts/resource_alerts.libsonnet

```diff
@@ -156,6 +158,20 @@
               summary: 'Processes experience elevated CPU throttling.',
             },
           },
+          {
+            alert: 'KubePodEvictionRateHigh',
+            expr: |||
+              sum(rate(kubelet_evictions[15m])) > %(highEvictionRateThreshold)s
```

Comment:
Did you check [kube_pod_status_reason](https://github.com/kubernetes/kube-state-metrics/blob/master/docs/pod-metrics.md) ? I think the `reason` label of this metric shall be used to find whther the pod has been evicted.

Reply:
Hmm looks like that metric can't be used to determine the eviction rate since the query returns results for hours/days after the eviction actually happened (those two evictions happened yesterday, but appear when searching for evictions in the last 15 minutes):
<img width="1501" alt="image" src="https://user-images.githubusercontent.com/7581860/162990007-a87337ae-241c-496c-bf66-581870dd46b4.png">

Maybe there's a way we could still use `kube_pod_status_reason` query results in the description? Or is the description limited to using results from the `expr`?